src/routes/documents/+page.svelte

```diff
@@ -54,12 +61,14 @@
   <DocumentBrowser slot="content" documents={searchResults} {query} {pending} />
 
   <svelte:fragment slot="action">
-    <SignedIn>
-      <Button mode="primary" href="/upload/">
-        <PlusCircle16 />{$_("sidebar.upload")}
-      </Button>
+    {#if isSignedIn($me)}
```

Comment:
I think this nesting is fine. We have two things we need to check, and one is going to be contingent on the other. Just a side effect of the system we have.